synchronizer/actions/etrog/processor_l1_info_tree_update.go

```diff
@@ -24,9 +24,9 @@ type ProcessorL1InfoTreeUpdate struct {
 // NewProcessorL1InfoTreeUpdate new processor for GlobalExitRootsOrder
 func NewProcessorL1InfoTreeUpdate(state stateProcessorL1InfoTreeInterface) *ProcessorL1InfoTreeUpdate {
 	return &ProcessorL1InfoTreeUpdate{
-		ProcessorBase: actions.ProcessorBase[ProcessorL1InfoTreeUpdate]{
-			SupportedEvent:    []etherman.EventOrder{etherman.L1InfoTreeOrder},
-			SupportedForkdIds: &actions.ForksIdAll},
+		ProcessorBase: *actions.NewProcessorBase[ProcessorL1InfoTreeUpdate](
```

Comment:
What is the reason for this change?

Reply:
Just a refactoring to match the way I understood the `ProcessorBase` struct was intended to be implemented.
There were some methods to expose the fields adding extra logic when accessing these fields, but the fields were public and, there was no constructor to the struct, it was mixed publicly and privately implemented, then I adjusted it to work in a private way. 

This was done in the middle of other changes to the code, we can review it and change it if necessary.